client/verta/verta/_cli/registry/get.py

```diff
@@ -27,7 +33,20 @@ def get():
 def get_model(model_name, output, workspace):
     """Get detailed information about a model.
     """
-    pass
+    client = Client()
```

Comment:
@conradoverta This prints all the usual `set * from environment` messages. Should these prints be suppressed in the CLI?

Reply:
Yes, but can be done later. Ideally those messages are not `print` but with a logger so that we can control that better. For later.